omod-2.0/src/test/resources/nameTemplate.xml

```diff
@@ -0,0 +1,15 @@
+<org.openmrs.module.webservices.rest.web.v1_0.template.openmrs2_0.NameTemplate2_0>
```

Comment:
Why are we using `NameTemplate2_0` instead of `NameTemplate`?

Reply:
I had not seen the `NameTemplate` but I rectified this in the last commit